src/resmom/start_exec.c

```diff
@@ -3619,6 +3619,12 @@ finish_exec(job *pjob)
 		/*		streams to a socket connected to qsub.			 */
 		/*************************************************************************/
 
+		/* prevent user from interrupting start of the job */
+		signal(SIGHUP, SIG_IGN);
+		signal(SIGINT, SIG_IGN);
+		signal(SIGQUIT, SIG_IGN);
+		signal(SIGTSTP, SIG_IGN);
```

Comment:
If you use sigaction(), you can have it return you the current signal handler.  You can then set it back later in finish_exec().  This way if we ever decide to set a handler for one of these 4 signals at the start of mom, you don't clear it here.